source/ed/build_helper.cpp

```diff
@@ -760,6 +760,7 @@ void builder::make() {
     data->pt_data->sort_and_index();
     data->build_uri();
     finish();
+    data->build_relations();
```

Comment:
Why do you need that? (And what's the impact on the whole test times)

If you add it, then you should remove the calls in tests that are currently doing `make()` + `
build_relations()`

Reply:
Ok done